server/src/main/java/org/opensearch/cluster/metadata/Metadata.java

```diff
@@ -179,13 +179,30 @@ public interface Custom extends NamedDiffable<Custom>, ToXContentFragment, Clust
         EnumSet.of(ClusterBlockLevel.WRITE, ClusterBlockLevel.METADATA_WRITE)
     );
 
+    public static final ClusterBlock CLUSTER_CREATE_INDEX_BLOCK = new ClusterBlock(
+        10,
```

Comment:
What is the significance of this value `10`?

Reply:
Cannot find any specific significance of 10. It is just a id associated with a cluster block.